test/webaudio/audioworklet_test_shared.inc

```diff
@@ -5,18 +5,32 @@
 // Helper for MEMORY64 to cast a void* to an audio context or type
 #define VOIDP_2_WA(ptr) ((EMSCRIPTEN_WEBAUDIO_T) (intptr_t) ptr)
 
-// Count the audio callbacks and return after 375 frames (1 second with the default 128 size)
+// Attribute that marks a function, parameter, variable, etc., as unused
+// (silencing any -Wunused warnings).
+#ifndef __unused
+#define __unused __attribute__((unused))
+#endif
```

Comment:
I don't remember having to do this for other tests.   Are these tests build with different warnings enabled?

Maybe just build with `-Wno-unused` to keep the code simple?

Reply:
I wrote these tests standalone and I'm used to working as `-Wpedantic`.  I can take these out, it'll build without.